django_cyverse_auth/models.py

```diff
@@ -224,7 +223,7 @@ def lookupSessionToken(request):
     """
```

Comment:
It is just because you changed the function name from `AuthToken` to `Token`. And I think it  would be great to change the docs string as well. It is ok for you not to do it. I could do it.